openvpn/start.sh

```diff
@@ -4,9 +4,12 @@ if [ "$OPENVPN_PROVIDER" = "BTGUARD" ]
 then
 	echo "VPN PROVIDER: BTGUARD"
 	vpn_provider="btguard"
-else
+else if [ "$OPENVPN_PROVIDER" = "PIA" ]
```

Comment:
I agree @dougnukem . Thinking something like:
if provider = btguard
else if provider = pia
else if ...
else echo "No valid provider given"; exit 1
fi

ish?
This will also be backward compatible as OPENVPN_PROVIDER is set to PIA in Dockerfile. Could of course remove the default and make provider a required input as well. Defaulting to PIA makes less sense when the number of providers grows...